app/controllers/facility_statements_controller.rb

```diff
@@ -29,6 +29,7 @@ def index
   def new_with_search
     @order_details = @order_details.need_statement(@facility)
     @order_detail_action = :send_statements
+    set_default_start_date if SettingsHelper.feature_on?(:set_start_date)
```

Comment:
Looks pretty good, but can we make the feature flag name a little more specific, so we know it applies to the statementing search?